crates/rspack_plugin_javascript/src/plugin/mod.rs

```diff
@@ -311,22 +315,34 @@ impl JsPlugin {
         sources.add(RawSource::from("return __webpack_exports__;\n"));
       }
     }
-    let mut final_source = if compilation.options.output.iife {
-      self.render_iife(sources.boxed())
-    } else {
-      sources.boxed()
-    };
+    if is_iife {
+      sources.add(RawSource::from("\n})()"));
```

Comment:
The change is not expected, the pervious implmentemt is ok.

Reply:
https://github.com/webpack/webpack/blob/f87ff767461beaa00210e6d7d2e95af94b7982c8/lib/javascript/JavascriptModulesPlugin.js#LL942C1-L952C66

webpack is webpack is added after render "; \n”

If I had done this before I would have generated this error code
```
System.register([], function(__WEBPACK_DYNAMIC_EXPORT__, __system_context__) {
return {
execute: function() {
__WEBPACK_DYNAMIC_EXPORT__((function() {
...xxx

})();)}
//   ^ error 
}
});

```

Reply:
@suxin2017 I remove `render_iife` `;`, it worked fine.

Reply:
👌